main.py

```diff
@@ -24,38 +24,24 @@
 dispatcher = updater.dispatcher
 lad_bot = updater.bot
 
-quote_msgs = ['Here\'s some good quotery for you',\
-'Here\'s a nice quote for you', 'Here you go',\
-'Here you are', 'This is a nice one',\
-'I like this one', 'This one really resonated with me',\
-'This one\'s cute', 'This is a cute one',\
-'Here\'s a cute quote', 'This one\'s pretty cool',\
-'Here\'s a nice quote for you', 'Here you go',\
-'Here you are', 'This is a nice quote',\
-'I like this quote', 'This quote really resonated with me',\
-'This quote\'s cute', 'This is a cute quote',\
-'Here\'s a cute quote', 'This is a cool quote',\
-'I think you\'ll like this one', 'I think you\'ll like this quote']
-
-song_msgs = ['Here\'s some good songery for you',\
-'Here\'s a nice song for you', 'Here you go',\
-'Here you are', 'This is a nice one',\
-'I like this one', 'This one really resonated with me',\
-'This one\'s cute', 'This is a cute one',\
-'Here\'s a cute song', 'This one\'s pretty cool',\
-'Here\'s a nice song for you', 'Here you go',\
-'Here you are', 'This is a nice song',\
-'I like this song', 'This song really resonated with me',\
-'This song\'s cute', 'This is a cute song',\
-'Here\'s a cute song', 'This is a cool song',\
-'I think you\'ll like this one', 'I think you\'ll like this song']
+msgs = ["Here's a nice _ for you", "Here you go",
+"Here you are", "This is a nice one",
+"I like this one", "This one really resonated with me",
+"This one's cute", "This is a cute one",
+"Here's a cute _", "This one's pretty cool",
+"Here's a nice _ for you", "Here you go",
+"Here you are", "This is a nice _",
+"I like this _", "This _ really resonated with me",
+"This _'s cute", "This is a cute _",
+"Here's a cute _", "This is a cool _",
+"I think you'll like this one", "I think you'll like this _"]
 
 
 def inspire(update, context):
     """Send a quote"""
 
     quote = quotes.get_quote()
-    lad_bot.send_message(chat_id=update.effective_chat.id, text=f"{r.choice(quote_msgs)}, {update.message.from_user.first_name}:", disable_notification=True)
+    lad_bot.send_message(chat_id=update.effective_chat.id, text=f"{r.choice(msgs).replace('_', 'quote')}, {update.message.from_user.first_name}:", disable_notification=True)
```

Comment:
Adhere to the 120 character limit set by PEP-8. Scrolling sideways while reviewing code is the last thing I want to do